ifrs17/Import/ImportStorage.ipynb

```diff
@@ -319,6 +319,9 @@
                 "\n",
                 "\n    public DataNodeData GetDataNodeData(ImportIdentity id) => DataNodeDataBySystemName[id.DataNode];",
                 "\n",
+                "\n    public SingleDataNodeParameter GetSingleDataNodeParameter(ImportIdentity id, int period) => SingleDataNodeParametersByGoc.TryGetValue(id.DataNode, out var singleDataNodeParameter)",
+                "\n            ? singleDataNodeParameter[period] : default;",
```

Comment:
Log an error instead of returning default.

Reply:
addded a validation